Documentation/development-flow.md

```diff
@@ -277,3 +277,28 @@ git push origin HEAD
 
 The last step is to open a PR with the base being the intended release branch.
 Once the PR is approved and merged, then your backported change will be available in the next release.
+
+
+## Debugging operators locally
+
+Operators are meant to be run inside a kubernetes cluster. However, this makes it harder to use debugging tools and slows down the developer cycle of edit-build-test since testing requires to build a container image, push to the cluster, restart the pods, get logs, etc.
```

Comment:
```suggestion
Operators are meant to be run inside a Kubernetes cluster. However, this makes it harder to use debugging tools and slows down the developer cycle of edit-build-test since testing requires to build a container image, push to the cluster, restart the pods, get logs, etc.
```